Doc/library/os.rst

```diff
@@ -569,6 +569,28 @@ process and user.
       See the documentation for :func:`getgroups` for cases where it may not
       return the same group list set by calling setgroups().
 
+.. function:: setns(fd, nstype=0)
+
+   Reassociate thread with a namespace.
+   If *fd* refers to a ``/proc/[pid]/ns/`` link, :func:`setns` reassociates the
+   calling thread with the namespace associated with that link, subject to any
+   constraints imposed by the *nstype* argument (or any if 0).
+   Since Linux 5.8, *fd* may refer to a PID file descriptor obtained from
+   :c:func:`pidfd_open`. In this case :func:`setns` reassociates the calling thread
+   into one or more of the same namespaces as the thread referred to by *fd* subject
+   to any constraints imposed by the *nstype* which is a bit mask specified by ORing together one or more of the ``CLONE_NEW*`` constants. the caller's memberships in unspecified namespaces are left unchanged.
```

Comment:
Could you please wrap all lines within 79 characters? Also, using semantic breaks will help reduce future diffs.

Reply:
Also, instead of ``ORing together``, I'd suggest something like this:

```
a bit mask specified by combining one or more or the ``CLONE_NEW*`` constants using ``|`` (bitwise or).
```

Ditto for the ``ORing`` below.